### PR TITLE
Update filters.md

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: aspnetcore/mvc/controllers/filters.md [hunk 1]
```diff
@@ -538,6 +538,14 @@ For example, the following filter always runs and sets an action result (<xref:M
...
 
+IFilterFactory.IsReusable` is a hint by the factory that the filter instance created by the factory may be reused outside of the request scope it was created within. The ASP.NET Core runtime doesn't guarantee:
+
+* That a single instance of the filter will be created.
+* The filter will not be re-requested from the DI container at some later point.
+* Should not be used with a filter that depends on services with a lifetime other than singleton.
+
+[!WARNING] Only configure `IFilterFactory.IsReusable` to return `true` if the source of the filters is unambigious, the filters are stateless, and are safe to use across multiple HTTP requests.
```
Comment (davidfowl) [inline aspnetcore/mvc/controllers/filters.md:547]: I would say something specifically about not using scoped services.

Comment (pranavkm) [reply aspnetcore/mvc/controllers/filters.md:547]:
```suggestion
[!WARNING] Only configure `IFilterFactory.IsReusable` to return `true` if the source of the filters is unambigious, the filters are stateless, and are safe to use across multiple HTTP requests. For instance, do return filters from DI that are registered as scoped or transient if `IFilterFactory.IsReusable` returns `true`
```

Comment (davidfowl) [reply aspnetcore/mvc/controllers/filters.md:547]: do *NOT*

Comment (pranavkm) [reply aspnetcore/mvc/controllers/filters.md:547]: oops. Fixed

**Comment by pranavkm** [comment]
@Rick-Anderson could you review? Feel free to merge it.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**